earth_enterprise/rpms/build.gradle

```diff
@@ -487,6 +487,8 @@ task openGeeServerRpm (type: GeeRpm, dependsOn: openGeeSharedFiles) {
     // `initscripts` package:
     requires('initscripts')
     requires('python-unittest2')
+    // The WMS Service requires PIL, which is not automatically detected:
+    requires('python-imaging')
```

Comment:
We should perhaps add something to the build or install instructions to indicate that this is required.

Reply:
Hm. It seems like we may need some documentation clean up. The current install instructions don't seem to specify what packages are need at run time, and you don't need this package to build Open GEE.

Reply:
Off the top of my head, perhaps we could add a section to the install documentation that tells what libraries to install for run time support. That might get to be a big effort though - you could start on it on this ticket if you like, or we could write a follow up.